doc/man3/RAND_set_rand_method.pod

```diff
@@ -58,7 +60,7 @@ L<RAND_bytes(3)>, L<ENGINE_by_id(3)>
 
 =head1 COPYRIGHT
 
-Copyright 2000-2017 The OpenSSL Project Authors. All Rights Reserved.
+Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Why did you change 2000 to 2017?  If you `git log --stat --follow -C -M`, you will see that this page was, in fact, started in 2000.

Reply:
Just noticed that the year was still wrong and pushed a fixup.